helix-view/src/lib.rs

```diff
@@ -68,3 +93,26 @@ pub use document::Document;
 pub use editor::Editor;
 pub use theme::Theme;
 pub use view::View;
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    #[test]
+    fn test_view_index_to_identifier_and_back() {
+        assert_eq!(view_index_to_identifier(0), Some('1'));
+        assert_eq!(view_index_to_identifier(8), Some('9'));
+        assert_eq!(view_index_to_identifier(9), Some('a'));
+        assert_eq!(view_index_to_identifier(34), Some('z'));
+        assert_eq!(view_index_to_identifier(35), None);
+        assert_eq!(view_identifier_to_index('0'), None);
+        assert_eq!(view_identifier_to_index('1'), Some(0));
+        assert_eq!(view_identifier_to_index('9'), Some(8));
+        assert_eq!(view_identifier_to_index('a'), Some(9));
+        assert_eq!(view_identifier_to_index('z'), Some(34));
+        assert_eq!(view_identifier_to_index('A'), Some(9));
+        assert_eq!(view_identifier_to_index('Z'), Some(34));
```

Comment:
Lowercase and uppercase map to the same view index? I think that's surprising behavior

Reply:
i would not think that any user with have more than 34 windows up in practice, so therefore i thought there would be no need to have different windows for 0-9, a-z and then A-Z. Therefore I thought that it would be nice to make this case insensitive. However it may be cleaner to just handle lower case keys (and ignore capital letters). I can refactor the code to act this way.